CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -666,16 +761,16 @@ protected override bool IsInputKey(Keys keyData)
                 case Keys.Up:
                 case Keys.Down:
                 case Keys.Tab:
-                { 
```

Comment:
> VS uses tabs by default for C# files unless you've changed the default.

I can't remember changing that and mine uses spaces

> GitHub still thinks the formatting has changed, usually it's when you've used Tabs instead of spaces.

I swear it's spaces